### ISSUE TITLE
CMS data tables using but scope and header attributes

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10295]
## Description
A couple tables in the Vet Center product within the CMS use both scope and header attributes which may make it difficult for screen readers to read the table data correctly.

## Additional Context
List of tables where this was found (may not be the full list of all tables within the CMS):

- Vet Center Dashboard: Satellite Locations table
- Vet Center - Locations List Edit screen: Main and Satellite Locations Table

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/184972571-ef686ff4-ecad-4a2f-adab-b5917284b904.png)

## Accessibility Standard

WCAG version 2.0 A, [Criterion 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

## Acceptance Criteria
- [ ] Technical review for removing header attributes from tables (one update for all or individual updates)
- [ ] Change management consulted
- [ ] Implementation ticket(s) created

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by srancour** [comment]
This is something that still exists and relates to my [comments](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/17755#issuecomment-2073551085) on #17718

**Comment by gracekretschmer-metrostar** [comment]
Scope of impact: breaks the experience in the screen readers for table headings and, in turn, end users cannot understand the contents of a table.
